### DISCUSSION TITLE
Wireguard Dashboard

### DISCUSSION THREAD

**Discussion by imellor** [tteck/Proxmox#3332] [General] [answered]
Hi All,
I'm relatively new to Proxmox scripts, last month I downloaded Wireguard and it included a dashboard. I recently reinstalled, but this now appears to be no longer part of the script?

**Comment by tteck** [comment, marked as answer]
![Screenshot from 2024-07-05 17-38-15](https://github.com/tteck/Proxmox/assets/21342670/82b3d55a-5a78-4c9d-a9a6-eebc716e2d72)

**Reply by imellor** [reply]
Yes that is the screen I remember, that is no longer on the website.
![image](https://github.com/tteck/Proxmox/assets/53119550/f580a7b9-3882-46ca-8126-d6a333bcfa5e)

**Reply by tteck** [reply]
lol, read it again

**Reply by imellor** [reply]
Oh, sorry. I'm in the UK and there is an advert that says "Should have gone to Specsavers" I think that applies here :-)
